docs/tailwind.config.js

```diff
@@ -1,20 +1,8 @@
 module.exports = {
-  theme: {
-    extend: {
-      colors: {
-        primary: {
-          50: '#F2F9F7',
-          100: '#E6F3F0',
-          200: '#C0E2D8',
-          300: '#9AD0C1',
-          400: '#4EAD93',
-          500: '#028A64',
-          600: '#027C5A',
-          700: '#01533C',
-          800: '#013E2D',
-          900: '#01291E'
-        }
-      }
-    }
+  future: {
```

Comment:
Idea: Maybe we should add these flags to theme?

Reply:
Will make a PR to add the news flags: https://github.com/nuxt/content/blob/dev/packages/theme-docs/src/tailwind.config.js#L15-L16

Reply:
PR live on https://github.com/nuxt/content/pull/573